stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
I'm not even importing them all. If it's necessary, the build will fail and we can fix it. Also, I'm about to be able to test cmake configurations quickly!